src/helpers/page/WrapSetTimeoutInPromise.ts

```diff
@@ -0,0 +1,13 @@
+
+export async function wrapSetTimeoutInPromise(callback: Function, timeDelaySeconds: number): Promise<void> {
+    await new Promise((resolve, reject) => {
+    setTimeout(async () => {
+        try {
+            await callback();
+            resolve();
+        } catch(e) {
+            reject(e);
```

Comment:
Don't think a catch and reject is needed here. If the callback throws it will bubble up in the same way I believe.